Text/LaTeX/Packages/Acronym.hs

```diff
@@ -24,7 +24,7 @@ module Text.LaTeX.Packages.Acronym
 import Data.String(IsString(fromString))
 
 import Text.LaTeX.Base.Class(LaTeXC, comm0, comm1, comm2, liftL, liftL2)
-import Text.LaTeX.Base.Syntax(LaTeX(TeXComm, TeXEnv), TeXArg(FixArg, OptArg))
```

Comment:
Maybe this module, and the other `Package` ones, could simply be changed to not rely on plain constructors in the first place. Then there wouldn't be a need for fiddling around with pattern synonyms.